ee/clickhouse/views/cohort.py

```diff
@@ -35,6 +48,63 @@ def insert_cohort_people_into_pg(cohort: Cohort):
     cohort.insert_users_list_by_uuid(items=[str(id[0]) for id in ids])
 
 
+def insert_cohort_actors_into_ch(cohort: Cohort, filter_data: Dict):
+    insight_type = filter_data.get("insight")
+
+    if insight_type == INSIGHT_TRENDS:
+        filter = Filter(data=filter_data)
+        entity = get_target_entity(filter)
+        query, params = ClickhouseTrendsActors(cohort.team, entity, filter, limit_actors=False).actor_query()
+    elif insight_type == INSIGHT_STICKINESS:
+        stickiness_filter = StickinessFilter(data=filter_data, team=cohort.team)
+        entity = get_target_entity(stickiness_filter)
+        query, params = ClickhouseStickinessActors(
+            cohort.team, entity, stickiness_filter, limit_actors=False
+        ).actor_query()
+    elif insight_type == INSIGHT_FUNNELS:
+        funnel_filter = Filter(data=filter_data)
```

Comment:
Bug: This doesn't simplify, meaning test account filters don't get accounted for. It probably should.